Index: work_moved_flask-admin-1.5.4-py_0_noarch/flask_admin/contrib/fileadmin/__init__.py
===================================================================
--- work_moved_flask-admin-1.5.4-py_0_noarch.orig/flask_admin/contrib/fileadmin/__init__.py
+++ work_moved_flask-admin-1.5.4-py_0_noarch/flask_admin/contrib/fileadmin/__init__.py
@@ -8,7 +8,11 @@ import shutil
 from operator import itemgetter
 
 from flask import flash, redirect, abort, request, send_file
-from werkzeug import secure_filename
+try:
+    from werkzeug import secure_filename
+except ImportError:
+    from werkzeug.utils import secure_filename
+
 from wtforms import fields, validators
 
 from flask_admin import form, helpers
Index: work_moved_flask-admin-1.5.4-py_0_noarch/flask_admin/form/upload.py
===================================================================
--- work_moved_flask-admin-1.5.4-py_0_noarch.orig/flask_admin/form/upload.py
+++ work_moved_flask-admin-1.5.4-py_0_noarch/flask_admin/form/upload.py
@@ -1,7 +1,11 @@
 import os
 import os.path as op
 
-from werkzeug import secure_filename
+try:
+    from werkzeug import secure_filename
+except ImportError:
+    from werkzeug.utils import secure_filename
+
 from werkzeug.datastructures import FileStorage
 
 from wtforms import ValidationError, fields
Index: work_moved_flask-admin-1.5.4-py_0_noarch/flask_admin/model/base.py
===================================================================
--- work_moved_flask-admin-1.5.4-py_0_noarch.orig/flask_admin/model/base.py
+++ work_moved_flask-admin-1.5.4-py_0_noarch/flask_admin/model/base.py
@@ -5,7 +5,10 @@ import mimetypes
 import time
 from math import ceil
 
-from werkzeug import secure_filename
+try:
+    from werkzeug import secure_filename
+except ImportError:
+    from werkzeug.utils import secure_filename
 
 from flask import (current_app, request, redirect, flash, abort, json,
                    Response, get_flashed_messages, stream_with_context)
